### ISSUE TITLE
AssertionError on a binary diff

### ISSUE THREAD

**Issue by abbradar** [cscorley/whatthepatch#56]
With [this Git binary diff](https://gist.github.com/abbradar/d2981f1c5f64b54fcc968da5cc1973cd), I get this error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/nix/store/6qz1fvnb18gzydhm8awy94lrxb0sy2y5-python3.10-whatthepatch-1.0.4/lib/python3.10/site-packages/whatthepatch/patch.py", line 102, in parse_patch
    d = parse_diff(diff)
  File "/nix/store/6qz1fvnb18gzydhm8awy94lrxb0sy2y5-python3.10-whatthepatch-1.0.4/lib/python3.10/site-packages/whatthepatch/patch.py", line 199, in parse_diff
    return parser(lines)
  File "/nix/store/6qz1fvnb18gzydhm8awy94lrxb0sy2y5-python3.10-whatthepatch-1.0.4/lib/python3.10/site-packages/whatthepatch/patch.py", line 995, in parse_git_binary_diff
    assert len(line) >= 7 and ((len(line) - 2) % 5) != 0
AssertionError
```

This issue has originally been encountered in [Ki](https://github.com/langfield/ki/issues/127).

**Comment by babenek** [comment]
Confirmed.
The fix in PR #57

**Comment by cscorley** [comment]
Thanks for the help :)
